otter/test/log/test_spec.py

```diff
@@ -171,3 +176,104 @@ def test_callable_spec_split_events(self):
               'otter_msg_type': 'foo-bar',
               'ab': 'cd',
               'split_message': '2 of 2'}])
+
+
+class ExecuteConvergenceSplitTests(SynchronousTestCase):
+    """
+    Tests for splitting "execute-convergence" type events
+    (e.g. :func:`split_execute_convergence`)
+    """
+    def test_split_out_servers_if_servers_longer(self):
+        """
+        If the 'servers' parameter is longer than the 'lb_nodes' parameter,
+        and the event is otherwise sufficiently small, 'servers' is the
+        param that gets split into another message.
+        """
+        event = {'hi': 'there', 'desired': 'desired', 'steps': ['steps'],
+                 'lb_nodes': ['1', '2', '3'], 'servers': ['1', '2', '3', '4']}
+        message = "Executing convergence"
+
+        # assume that removing 'lb_nodes' would make it the perfect length, but
+        # since 'servers' is bigger, it's the thing that gets removed.
+        length = len(
+            json.dumps({k: event[k] for k in event if k != 'lb_nodes'}))
+
+        result = split_execute_convergence(event.copy(), max_length=length)
+        expected = [
+            ({k: event[k] for k in event if k != 'servers'}, message),
```

Comment:
Nitpick: You can use [`dissoc`](http://toolz.readthedocs.org/en/latest/api.html#toolz.dicttoolz.dissoc) in all these places: 
```python
(dissoc(event, "servers"), message),
(dissoc(event, "desired", "steps", "lb_nodes"), message)
```